reframe/core/pipeline.py

```diff
@@ -181,9 +181,9 @@ def disable_hook(cls, hook_name):
     valid_systems = fields.TypedField('valid_systems', typ.List[str])
 
     #: Logging of arbitrary information in per test basis
```

Comment:
Leave a new line after here.

Reply:
I removed completely the `log_extra` attribute. It's not needed. If the `foo` attribute in `%(check_foo)s` cannot be found, it will be simply logged as `None`.